### PR TITLE
LS-Dyna example

### PR DESCRIPTION
**Pull request by ayush-kumar-423**
LS-Dyna example

### CODE REVIEW

**Review by vnamdeo — APPROVED** [review]

File: .codespellrc [hunk 1]
```diff
@@ -1,4 +1,4 @@
 [codespell]
 skip = *.pyc,*.xml,*.txt,*.gif,*.png,*.jpg,*.js,*.html,*.doctree,*.ttf,*.woff,*.woff2,*.eot,*.mp4,*.inv,*.pickle,*.ipynb,flycheck*,./.git/*,./.hypothesis/*,*.yml,./doc/build/*,./doc/images/*,./dist/*,*~,.hypothesis*,./doc/source/examples/*,*cover,*.dat,*.mac
 quiet-level = 3
-ignore-words-list = DataModel
+ignore-words-list = DataModel, GlobalY
```
Comment (RobPasMue) [inline .codespellrc:4]:
```suggestion
ignore-words-list = DataModel,GlobalY
```

**Comment by vnamdeo** [comment]
@koubaa @samigithub2022 Debug further this example and noticed that no issue observed in documentation creation locally. However, when we pushed to git, Documentation step took 1+ hour's and still shows "In Progress" mode as @samigithub2022 mentioned. Could you please look in to the issue at the documentation workflow level. Thanks !

**Comment by koubaa** [comment]
@RobPasMue any idea why the html doc build would time out? I don't see anything in the logs

**Comment by RobPasMue** [comment]
> @RobPasMue any idea why the html doc build would time out? I don't see anything in the logs

No idea. I'd have to look into it. Let me try

**Comment by samigithub2022** [comment]
@vnamdeo @koubaa @RobPasMue - I have noticed - if Mechanical crashed, doc. was hanging. I will debug this issue.

**Comment by RobPasMue** [comment]
I am doing some remote debugging just in case... but it looks like the Mechanical session crashed indeed

**Comment by koubaa** [comment]
> @koubaa @samigithub2022 @vnamdeo @ayush-kumar-423 - Have a look at https://github.com/ansys/pymechanical-examples/actions/runs/5444683310/jobs/9902895691?pr=34
> 
> It looks like the script finishes successfully, but is not able to continue to the next example... something weird is happenning with the mechanical sessions IMO. But I can't really tell...

@samigithub2022 @RobPasMue probably worth trying to debug locally with docker compose (i.e. outside of github)

**Comment by samigithub2022** [comment]
@RobPasMue - thanks for your remote debugging .

@ayush-kumar-423 @vnamdeo @koubaa @RobPasMue  - I have debugged this.

FYI: We run our pipeline using 232 docker image.

It was hanging in analysis.Solver line.

For standalone - In 231 we had support for lsydnya and it stopped working in 232. In 241 also, it didn't work. I will file a defect on the development side to fix this issue.

We may not be able to support this example currently. We'll support it for 241. We'll file a defect on the github side to keep track of this issue.

When you create an example, if possible please test it also on Linux.

**Comment by samigithub2022** [comment]
FYI: @ayush-kumar-423 @vnamdeo @koubaa @RobPasMue 

An issue has been created to track this:

https://github.com/ansys/pymechanical-examples/issues/52

**Comment by ayush-kumar-423** [comment]
In discussion with @vnamdeo we can let this PR open till #52 is fixed and revisit afterwards.

**Comment by dipinknair** [comment]
> In discussion with @vnamdeo we can let this PR open till #52 is fixed and revisit afterwards. cc: @koubaa @samigithub2022
@ayush-kumar-423 @vnamdeo 
This will be fixed with 241 which will support full LSDyna functionalities.  Please see below results generated in 241.
image

**Comment by dipinknair** [comment]
Updates in progress

- [ ]  image export
- [ ]  animation export
- [ ] doc cleanup

**Review by dipinknair — APPROVED** [review]

File: examples/basic/example_08_lsdyna_taylor_bar.py [hunk 2]
```diff
@@ -0,0 +1,324 @@
...
+geometry_import.Import(part_file_path, geometry_import_format, None)
+
+Model.AddLSDynaAnalysis()
+analysis = Model.Analyses[0]
+
+ExtAPI.Application.ActiveUnitSystem = MechanicalUnitSystem.StandardNMMton
+ExtAPI.Application.ActiveAngleUnit = AngleUnitType.Radian
+
+MAT = ExtAPI.DataModel.Project.Model.Materials
+#MAT.Import(mat_file_path)
```
Comment (vnamdeo) [inline examples/basic/example_08_lsdyna_taylor_bar.py:97]: @dipinknair can you uncomment MAT.Import(mat_file_path)

File: examples/basic/example_08_lsdyna_taylor_bar.py [hunk 3]
```diff
@@ -0,0 +1,324 @@
...
+
+# Assign the material
+#ExtAPI.DataModel.Project.Model.Geometry.Children[0].Children[0].Material = "Bullet"
```
Comment (vnamdeo) [inline examples/basic/example_08_lsdyna_taylor_bar.py:100]: @dipinknair can you assign material property "Bullet" instead of default steel.

**Review by vnamdeo — COMMENTED** [review]
@dipinknair I have added couple of comments to include material import and assign it to the bodies instead of structural steel. Thanks !

File: examples/basic/example_08_lsdyna_taylor_bar.py (hunk 2 above)
Comment (dipinknair) [reply examples/basic/example_08_lsdyna_taylor_bar.py:97]: Yes #171

**Pull request merged**